src/model/transaction/MosaicMetadataTransaction.ts

```diff
@@ -51,8 +51,7 @@ export class MosaicMetadataTransaction extends Transaction {
      * @param scopedMetadataKey - Metadata key scoped to source, target and type.
```

Comment:
isn't this change breaking a existing code using typescript sdk?

Reply:
> isn't this change breaking a existing code using typescript sdk?

Yes, it is. 
I tried to keep the implications minimal. Most probably in the clients how a metadata tx is created won't change if they use https://github.com/symbol/symbol-sdk-typescript-javascript/blob/079297ddc765c61a058b0d132afbe8c8dd826c6f/src/service/MetadataTransactionService.ts#L60 as we do in our clients(desktop-wallet, symbol-cli).
But the value type used for presentation will change ofc.